source/install/WSL.rst

```diff
@@ -0,0 +1,86 @@
+Setting up Windows to use QIIME 2
+---------------------------------
+
+This section will demonstrate how to set up windows to allow QIIME 2 installation. Windows now allows users to install linux as a windows subsystem (WSL), the simplest way to use QIIME 2 on a machine with Windows OS is therefore to use this Windows subsystem for Linux.
+
+Please see the `official windows tutorial <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`__ to install WSL2. Then install an appropriate linux distribution e.g. the latest stable version of Ubuntu. Once installation is complete run ``sudo apt-get update`` and ``sudo apt-get upgrade`` to ensure everything is up to date.
+
+Then install miniconda (or alternatively you could install Anaconda) using the following commands:
+
+.. command-block::
+    :no-exec:
+    
+    wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+    bash Miniconda3-latest-Linux-x86_64.sh
+    #once installation has completed
+    conda init 
+
+Restart the terminal to have changes take effect. Ensure you have the latest conda verstion and then create an environment to work in as follows:
+
+.. command-block::
+    :no-exec:
+    
+    conda update conda
+    conda create --name <name of new environment>
+
+Packages for your project can now be installed in this environment, keeping them separate from any other projects you may have, and ensuring that different dependencies do not clash. For instance, QIIME 2 can be installed into this environment as follows:
+
+Navigate to :doc:`Natively installing QIIME 2 <native>` in these docs to find the link for the latest version of QIIME 2. Use this to download QIIME 2:
+
+.. command-block::
+    :no-exec:
+
+    wget <link from QIIME 2 docs>
+    conda env update --file <RELEASE-NAME>.yml
+
+Or install QIIME 2 into a new environment as follows:
+
+.. command-block::
+    :no-exec:
+
+    wget <link from QIIME2 docs>
+    conda env create -n <name of new environment> --file <RELEASENAME>.yml
+    conda activate <name of new environment>
```

Comment:
Let's drop this section, since it is mostly just duplicating the "Native Install" docs. I think keeping each install guide a modular component will pay off well here.